src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Contains additional information relevant to a Deadline task.
+ * 
+ * @author Siau Wee
+ */
+public class Deadline extends Task {
+
+    private final DateTimeFormatter FORMAT_TIME_YYYYMMDD = 
+            DateTimeFormatter.ofPattern("HH:mm 'on' yyyy/MM/dd");
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMAT_TIME_YYYYMMDD = 
            DateTimeFormatter.ofPattern("HH:mm 'on' yyyy/MM/dd");
```

It would be more appropriate to have this constant be a `static` variable.